src/rust/ensogl/src/display/symbol/gpu.rs

```diff
@@ -435,12 +433,9 @@ impl Symbol {
         self.with_program_mut(|this,program|{
             for binding in var_bindings {
                 match binding.scope.as_ref() {
-                    Some(ScopeType::Mesh(s)) =>
-                        this.init_attribute_binding(program,binding,*s),
-                    Some(_) =>
-                        this.init_uniform_binding(program,binding,&mut texture_unit_iter),
-                    None =>
-                        {}
+                    Some(ScopeType::Mesh(s)) => this.init_attribute_binding(program,binding,*s),
+                    Some(_) => this.init_uniform_binding(program,binding,&mut texture_unit_iter),
+                    None    => {}
```

Comment:
Alignment.

Reply:
The aligment was broken only not to use multiline expressions here.